src/databases/VTK/PVTKParser.C

```diff
@@ -124,13 +124,18 @@ PVTKParser::SetFileName(const char *fn)
 //  Programmer: Kathleen Biagas
 //  Creation:   September 21, 2017
 //
+//  Modifications:
+//   Cyrus Harrison, Wed Apr 17 12:48:31 PDT 2019
+//   Change valid check, size_t is unsigned, no need to check for lower bound
+//
 // ****************************************************************************
 
 string &
 PVTKParser::GetPieceFileName(size_t index)
 {
     pieceName.clear();
-    if(index >=0 && index < pieceFileNames.size())
+    // index is unsigned, it will always be >=0
```

Comment:
These  comments are useful to us, now, when reviewing this particular PR related to compilation warnings but are otherwise noise I think and won't make sense for anyone arriving here in the future because they won't have the context. I think just don't include them.

Reply:
I like more comments :-)

Reply:
Sure...but not when a comment is referring to code that was removed. The reason for mentioning `it will always be >=0`  is that is what the old logic was. That logic is gone now and the type is unsigned. So, if I am wondering here with none of that context, I have no idea why that comment is, basically, telling me what unsigned means, which I should probably already know.

Reply:
i think the comment is still valid (applies to current code) - I agree it is odd.
Another reason why I added it is that I wasn't sure if this was part of borrowed code from VTK. It doesn't appear to be, in those cases having some context when you diff is helpful.